frontend/views/pages/Payments.vue

```diff
@@ -90,15 +117,23 @@ page(
               @change-page='handlePageChange'
               @change-rows-per-page='handleRowsPerPageChange'
             )
+
+        .c-container(v-else-if='ephemeral.activeTab === "PaymentRowTodo" && ephemeral.paymentMethodFilter === "lightning"')
+          p.c-lightning-todo-msg 'TODO: implementing Lightning payments.'
```

Comment:
Can you replace this with:

```pug
p.c-lightning-todo-msg Coming Soon.
```

And also only show the `Open Placeholder Modal` button if `process.env.NODE_ENV === 'development'` ?